src/main/java/org/jabref/model/pdf/search/PdfSearchResults.java

```diff
@@ -0,0 +1,45 @@
+package org.jabref.model.pdf.search;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public final class PdfSearchResults {
+
+    private final List<SearchResult> searchResults;
+
+    public PdfSearchResults(List<SearchResult> search) {
+        this.searchResults = Collections.unmodifiableList(search);
+    }
+
+    public PdfSearchResults() {
+        this.searchResults = Collections.unmodifiableList(Collections.emptyList());
+    }
+
+    public List<SearchResult> getSortedByScore() {
+        List<SearchResult> sortedList = new ArrayList<>(searchResults);
+        sortedList.sort((searchResult, t1) -> {
+            if (searchResult.getLuceneScore() < t1.getLuceneScore()) {
+                return -1;
+            }
+            if (searchResult.getLuceneScore() > t1.getLuceneScore()) {
+                return 1;
+            }
+            return 0;
+        });
```

Comment:
```suggestion
        sortedList.sort((searchResult, t1) -> Float.compare(searchResult.getLuceneScore(), t1.getLuceneScore()));
```

Suggested by IntelliJ 👀

Reply:
Wow. good catch, IntelliJ.